app/views/instructeurs/procedures/_header_field.html.haml

```diff
@@ -1,8 +1,9 @@
 %th{ class: classname }
   = link_to update_sort_instructeur_procedure_path(@procedure, table: field['table'], column: field['column']) do
-    = field['label']
     - if @procedure_presentation.sort['table'] == field['table'] && @procedure_presentation.sort['column'] == field['column']
       - if @procedure_presentation.sort['order'] == 'asc'
-        %img.caret-icon{ src: image_url("table/up_caret.svg") }
+        #{field['label']} ^
       - else
-        %img.caret-icon{ src: image_url("table/down_caret.svg") }
+        #{field['label']} v
```

Comment:
Mmm, pour moi l'icône en SVG est plus lisible qu'un caret `v`, je crois : je n'interprète pas vraiment le caret comme un sélecteur de tri, vu qu'il a une forme un peu différente.

Tu avais une raison particulière pour ce changement ?

Reply:
oui, tu peux le voir entre la 3ieme et 4image en description de la pr. Le fait d'utiliser tu texte séparé par espace insécable et non une image me permet d'éviter un retour à la ligne disgracieux lorsque l'utilisateur affiche beaucoup de colonne.

Néanmoins, je concède ce n'est pas le plus compréhensible ni accessible. Au départ, je pensais utiliser ↓ mais je n'ai pas trouvé d'information sur le support par les navigateurs de l'unicode 1.1 .

j'y tiens pas absolument et je peux retirer si besoin. Est ce que tu aurais une autre idée géniale ?

Reply:
En autre idée rapide, un `white-space: nowrap;` permet de ne pas séparer l'image du texte. (Mais toute la colonne deviendra nowrap, pas juste le dernier mot et l'image. Ça me semble acceptable, mais à voir).

Si on ne veut pas que la colonne soit no-wrap en entier, mais seulement le dernier mot, et qu'on se sent de peaufiner, on peut toujours faire un split, et générer un :

```html
<th>
  Nom du <span class="white-space: nowrap;">champ <img src="" /></span>
</th>
```

Reply:
Alors, j'ai vérifié, ↓ n'existe pas dans la font `muli` il va y avoir donc un fall back sur les fonts systems

ca marche pour à partir de 
- windows vista (2007, fin de support 2017)
- macos cheeta (2001, fin de support 2004)

et par android et ios.

Du coup, je me dis qu'on pourrait l'utiliser (avec un espace insécable devant). Est ce que ça t'irait ?

(source: https://www.granneman.com/webdev/coding/css/fonts-and-formatting/default-fonts et https://www.fileformat.info)

Reply:
Yep, bon pour moi !

ëtre compatible avec 2001 c'est classe. À ce rythme bientôt on sera rétro-compatibles avec les pyramides 👌